bulkRoleManager/bulkRoleManager.py

```diff
@@ -323,4 +483,15 @@ async def _draft_eligible_message(self, ctx):
         return await self.config.guild(ctx.guild).DraftEligibleMessage()
 
     async def _save_draft_eligible_message(self, ctx, message):
-        await self.config.guild(ctx.guild).DraftEligibleMessage.set(message)
\ No newline at end of file
+        await self.config.guild(ctx.guild).DraftEligibleMessage.set(message)
+
+    async def _send_member_message(self, ctx, member, message):
+        message_title = "**Message from {0}:**\n\n".format(ctx.guild.name)
+        command_prefix = await self._get_command_prefix(ctx)
```

Comment:
The prefix can be found from the context parameter. Instead of having a method to get it I'd suggest just using ctx.Prefix in the replace call below.

Reply:
ah I didn't know that. I will change this behavior, thanks!

Reply:
Done.